src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```
@@ -127,5 +127,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackTool.props" />
   <Import Project="$(MSBuildThisFileDirectory)Microsoft.NET.PackProjectTool.props" />
+
+  <Import Project="$(MSBuildThisFileDirectory)../../Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props" Condition="'$(_MicrosoftWindowsDesktopSdkImported)' != 'true'"/>
```

```suggestion
  <Import Sdk="Microsoft.NET.Sdk.WindowsDesktop"  Project="../targets/Microsoft.NET.Sdk.WindowsDesktop.props" Condition="'$(_MicrosoftWindowsDesktopSdkImported)' != 'true'"/>
```

This allows easier substitution of SDK via global json from a custom nuget feed. Can I ask why the above one wasn't used?

---

@Nirmal4G I don't know that we considered it.

---

@dsplaisted Can we do this then?

---

@dsplaisted Can we do this then?

---

What advantage that give us? I think it is easy to understand what a simple file import does.

---

Sure, but it does not allow substitute Sdks from a custom NuGet feed without patching the base SDK.